profiles/source/source_hdmiCEC.yml

```diff
@@ -4,3 +4,138 @@ hdmicec:
 
   features:
     extendedEnumsSupported: false
+
+
+cec_responses:
```

Comment:
move to seperate file under `assets/target`

these during the make process will be copied to the `bin` directory, and therefore will appear on the target as required during the test run.

This file should be rather than passing it via a command `run.sh` and add more -p knowing that it's in the bin directory.